test/org/sagebionetworks/bridge/stormpath/StormpathAccountTest.java

```diff
@@ -22,11 +24,14 @@
 import org.sagebionetworks.bridge.models.studies.StudyIdentifier;
 import org.sagebionetworks.bridge.models.studies.StudyIdentifierImpl;
 
+import com.newrelic.agent.deps.com.google.common.collect.Lists;
```

Comment:
We're importing the wrong class. Should we just remove new relic from our dependencies, since we don't use them anymore?